### DISCUSSION TITLE
Outdoor Air Temp. not updating

### DISCUSSION THREAD

**Discussion by octek0815** [raomin/ESPAltherma#306] [Q&A]
Hello,

I have indoor unit EHVH08S23EJ6V and outdoor unit ERGA06EAV3H and have an issue with "Outdoor air temp.(R1T)"
I am using definition "ALTHERMA(LT_CA_CB_04-08KW).h"

{0x20,0,105,2,1,"Outdoor air temp.(R1T)"}, is not updating and stays constantly on the same value.
Does anyone have an idea to solve the issue or am i using the wrong definition?
All other configured values are woking and also being updated.

**Comment by bugac** [comment]
same
![image](https://github.com/raomin/ESPAltherma/assets/404278/a3afc83a-f2ed-4e0c-85f7-d8559a6e9ffb)

**Reply by octek0815** [reply]
So I not alone. Just see that it updates when the outdoor unit is running.

**Comment by KristofKD** [comment]
Same for me. "Outdoor air temp" gets stuck as soon as I turn off the HP. 
The value must be somewhere, the ONECTA-App still shows it.

**Comment by Rablet** [comment]
I suspect this will be from having the power saving function enabled. I have the same on my EDLA04 unit and the functionality is described on page 58 of this doc: https://www.daikin-ce.com/content/dam/document-library/Installer-reference-guide/heat/air-to-water-heat-pump-low-temperature/EBLA04-08EV3.EBLA04-08E3V3.EDLA04-08EV3.EDLA04-08E3V3_Installer%20reference%20guide_4PEN685228-1B_English.pdf

_"When the power saving functionality of the outdoor unit is active (see "Power saving function"  [4212]), the outdoor unit is turned down to reduce standby energy losses. As a result, the outdoor ambient temperature is NOT read out."_

Something I have noticed is the external outside temperature sensor is reporting its values in "Ext. indoor ambient sensor (R6T)" and this is updated even when the pump is in standby mode.

**Reply by KristofKD** [reply]
This is interesting, thank you very much. I need to check this for my HP. At least the value is in the definition file.

**Reply by KristofKD** [reply]
Bummer, this gives me "0" all the time. But thanks anyway, it was worth a try.

**Reply by Rablet** [reply]
Ah annoying. Do you have an external temperature sensor? If you do you may have to enable it in the heat pump - mine was disabled as standard for some reason. I only noticed it when the temp would skyrocket when the HP itself was in the sun which made no sense given the external sensor was not in the sun. That's when I also discovered the R6T provided the outdoor temp for me.

**Reply by KristofKD** [reply]
An additional sensor device is on the way. It won't be compatible to the HP, but that is not necessary for me. It will be just one more device next to all the others ;-)

Besides that, I am delighted by the "update over the air". Crazy stuff!
